server/meca/routes.js

```diff
@@ -38,17 +38,17 @@ module.exports = app => {
     const updateStatus = () => Manuscript.updateStatus(manuscriptId, status)
 
     const sendEmail = () => mailer.send({
-      to: config.get('meca.notificationEmail'),
-      from: config.get('meca.fromAddressEmail'),
-      subject: 'MECA import failed',
+      to: config.get('meca.email.recipient'),
+      from: config.get('meca.email.sender'),
+      subject: `${config.get('meca.email.subjectPrefix')}MECA import failed`,
```

Comment:
Pedantic - but maybe put a space between the prefix and the the "MECA"?

Reply:
The space is in the config at the moment:
https://github.com/elifesciences/elife-xpub/blob/e59a7542692f7561126c3b29b37b26f48fc820c3/config/staging.js#L23

I think this is allow no prefix to be defined without having a leading space.

Reply:
In fact do we need this ... wouldn't it just make sense to pull it from `NODE_CONFIG_ENV` ?

Reply:
Then all the formatting (including the square braces) can be done in the subject line.

Reply:
Addressed in #1541 

There's a test for the subject field [here](https://github.com/elifesciences/elife-xpub/blob/1519-staging-email-refactor/server/xpub-server/entities/manuscript/resolvers/submitManuscript.test.js#L213-L216)